include/swift/Runtime/Config.h

```diff
@@ -359,7 +355,6 @@ extern uintptr_t __COMPATIBILITY_LIBRARIES_CANNOT_CHECK_THE_IS_SWIFT_BIT_DIRECTL
 #define __ptrauth_swift_escalation_notification_function
 #define __ptrauth_swift_dispatch_invoke_function
 #define __ptrauth_swift_accessible_function_record
-#define __ptrauth_swift_accessible_protocol_requirement_function_record
```

Comment:
we're removing the entire new section kind; this is safe since no release is using this yet.